internal/cloud/realm/app.go

```diff
@@ -11,10 +11,98 @@ import (
 )
 
 const (
-	appsPathPattern = adminAPI + "/groups/%s/apps"
-	appPathPattern  = appsPathPattern + "/%s"
+	appsPathPattern           = adminAPI + "/groups/%s/apps"
+	appPathPattern            = appsPathPattern + "/%s"
+	appDescriptionPathPattern = appPathPattern + "/description"
 )
 
+// DataSourceSummary is a short summary for a data source model
+type DataSourceSummary struct {
```

Comment:
[just a thought] might be nice to move this stuff to its own `app_description.go` file ... just reduce the overhead in this one file (it was already pretty hefty with the `Find` logic)